src/it/java/com/google/checkstyle/test/base/BaseCheckTestSupport.java

```diff
@@ -82,7 +82,7 @@ protected static Configuration getConfiguration() throws CheckstyleException {
      * @param clazz check class.
      * @return {@link DefaultConfiguration} instance.
      */
-    protected static DefaultConfiguration createCheckConfig(Class<?> clazz) {
+    private static DefaultConfiguration createCheckConfig(Class<?> clazz) {
```

Comment:
postponed till MethodDistance Check appear (base on our experiments in separate project).
I want to keep change simple in this commit to show reason why it become private.